Assets/Scripts/Strategy/BaseManagement/Buildings/Barracks.cs

```diff
@@ -61,6 +90,30 @@ public void CreateUnitEntry(IUnit unit)
             unitEntryObject.GetComponent<UnitEntryDisplay>().SetDisplay();
             unitEntryObject.GetComponent<ShowEntryModel>().Initialize(displayModelController.GetModel(unit.Role.Name), displayModelController.EnableModel);
             unitEntryObject.GetComponent<ShowEntryData>().Initialize(unit, displayDataController.SetData);
+            unitEntryObject.GetComponent<SetActiveEntry>().Initialize(SetActiveEntry);
+
+            return unitEntryObject;
+        }
+
+        /// <summary>
+        /// Sorts list of gameobject entries based off provided metric
+        /// </summary>
+        public void SortEntries(string metric)
+        {
+            foreach (Transform child in unitListPanel.transform)
+            { 
+                /*
+                 * Destroy all children
+                 */
+            }
+
+            /*
+             * Sort based off metric
+             */
+
+            /*
+             * Re-add children
+             */

```

Comment:
What is the point of this?

Reply:
I dont know why I added this now, but I figure down the road when there are a bunch of units you might want to sort based off role or something else